tests/common/data_access/test_data/fake_group_dao_data.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test data for group dao tests."""
+
+GET_GROUP_MEMBERS_SIDE_EFFECT = [
```

Comment:
Hmmm, there is no admin_directory_test.py, or at least not where I think it should be. The other test data for groups that we have is the fake_group.py and fake_group_members.py, and they are not structured to test for recursive membership, which the data here are.  And it kind of makes more sense to keep them here, closer to where it's being used in the group_dao.py.